harness/determined/workload.py

```diff
@@ -58,6 +58,9 @@ def from_json(dict: Dict[str, Any]) -> "Workload":
             dict["total_batches_processed"],
         )
 
+    def total_batch(self) -> int:
```

Comment:
Can we rename this to `target_batch()` maybe?  I don't think `total_batch()` communicates the right thing.